content/en/docs/concepts/security/pod-security-standards.md

```diff
@@ -280,20 +371,20 @@ mechanism.
 As mechanisms mature, they will be defined below on a per-policy basis. The methods of enforcement
 of individual policies are not defined here.
 
```

Comment:
also link to the PodSecurity page from here, since that's really where we want people to look first if they're just getting started, rather than PodSecurityPolicy?